daemons/dss-notify-v2/app.py

```diff
@@ -104,23 +105,25 @@ def launch_from_notification_queue(event, context):
         else:
             logger.warning(f"Recieved queue message with no matching subscription:{message}")
 
-def _notify_subscribers(replica: Replica, key: str, is_delete_event: bool):
+def handle_bundle_event(replica: Replica, key: str, is_delete_event: bool):
     if is_delete_event:
         metadata_document = get_deleted_bundle_metadata_document(replica, key)
     else:
-        if exists(replica, key):
-            metadata_document = record_event_for_bundle(replica, key)
-        else:
-            logger.error(f"Key %s not found in replica %s, unable to notify subscribers", key, replica.name)
-            return
+        metadata_document = build_bundle_metadata_document(replica, key)
+    if "CREATE" == metadata_document['event_type']:
+        record_event_for_bundle(replica, key)
```

Comment:
Should the metadata document that's returned by `record_event_for_bundle()` be saved to `metadata_document` variable? Or is it ignored b/c `record_event_for_bundle()` won't change the metadata document?